src/java.base/share/classes/java/lang/AutoCloseable.java

```diff
@@ -51,7 +52,11 @@
  * @author Josh Bloch
  * @since 1.7
  */
-@AnnotatedFor({"lock"})
+@AnnotatedFor({"lock", "mustcall", "nullness"})
+@CFComment({"MustCall:  Do not write @InheritableMustCall(close) because doing so requires",
+  "writing @InheritableMustCall({}) on Stream, and that leads to a `annotations.on.use` error at",
```

Comment:
@mernst is this `annotations.on.use` issue independent of the one fixed in https://github.com/typetools/checker-framework/pull/5334?